docs/studios/unity/unity-sdk/csharp/utilities.mdx

```diff
@@ -0,0 +1,300 @@
+```
+
+</TabItem>
+</Tabs>
+
+You can then destroy these spheres via their names, or simply destroy all spheres.
+
+**`public void DestroySphere(string name)`**
+
+**`public void DestroyAllSpheres()`**
+
+<Tabs>
+<TabItem value="delete_a_sphere" label="Delete a specific sphere" default>
+
+```csharp
+var thisPlayer = rgObject.FindPlayers()[0];
+RGGizmos.CreateSphere(thisPlayer.id, Color.green, 0.5f, true, "Player Sphere");
+
+// Later on...
+RGGizmos.DestroySphere("Player Sphere");
+```
+
+</TabItem>
+<TabItem value="delete_all_spheres" label="Delete all spheres" default>
+
+```csharp
+var thisPlayer = rgObject.FindPlayers()[0];
+RGGizmos.CreateSphere(thisPlayer.id, Color.green, 0.5f, true, "Player Sphere");
+
+// Later on...
+RGGizmos.DestroyAllSpheres();
+```
+</TabItem>
+</Tabs>
+
+
+### Drawing Text
+
+Drawing text is as simple as calling `RGGizmos.CreateText()` with the appropriate parameters. It allows you to draw
+text above an entity which always faces the camera.
+
+**`public void CreateText(int entityId, string text, float yOffset = 2.0f)`**
+
+Argument Name | Type | Default | Description
+---|---|---|---|
+`entityId` | `int` | required | The id of the `RGStateEntity` to draw the text above.
+`text`| `string` | required | The text to draw above the entity.
+`yOffset` | `float` | optional | The offset in the y direction to draw the text above the entity. Defaults to 2.
+
+<Tabs>
+<TabItem value="create_text" label="Text above entity" default>
+
+```csharp
+var thisPlayer = rgObject.FindPlayers()[0];
+RGGizmos.CreateText(thisPlayer.id, "Hello, World!");
+```
+
+</TabItem>
+<TabItem value="create_text_offset" label="Text above entity (with offset)" default>
+
+```csharp
+var thisPlayer = rgObject.FindPlayers()[0];
+RGGizmos.CreateText(thisPlayer.id, "Hello, World, but higher!", 5.0f);
+```
+
+</TabItem>
+</Tabs>
+
+You can then destroy this text via the entity id, or by deleting all of the texts drawn so far.
+
+**`public void DestroyText(int entityId)`**
+**`public void DestroyAllTexts()`**
+
+<Tabs>
+<TabItem value="delete_text" label="Delete text" default>
+
+```csharp
+var thisPlayer = rgObject.FindPlayers()[0];
+RGGizmos.CreateText(thisPlayer.id, "Hello, World!");
+
+// Later on...
+RGGizmos.DestroyText(thisPlayer.id);
+```
+
+</TabItem>
+
+<TabItem value="delete_all_text" label="Delete all text" default>
+
+```csharp
+var thisPlayer = rgObject.FindPlayers()[0];
+RGGizmos.CreateText(thisPlayer.id, "Hello, World!");
+
+// Later on...
+RGGizmos.DestroyAllTexts();
+```
+
+</TabItem>
+</Tabs>
+
+:::caution
+
+Because these billboard text objects are not using the Unity Gizmos framework, they will still be present in the scene
+if you create them and then turn off Gizmos. If you want to remove them, you must call `DestroyAllTexts()`.
+
+:::
+
+### Putting it all together
+
+Let's put this all together! Here is a snippet of code from the a bot written for Unity's Kart Racer sample. This demonstrates
```

Comment:
Should we add a link to the Kart Racer sample?

https://learn.unity.com/project/karting-template

Reply:
Fixed!